lib/Routes/reports.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+const dataService = require('../../models/data-service');
+const tableName = 'reports';
+const DebugDB = require('../../debuggers').server;
+const util = require('util');
+
+module.exports = (server) => {
+  server.post('/reports', async(req, res, next) => {
+    // get data from request body
+    const data = req.body;
+
+    DebugDB(`Get request data: ${util.inspect(data)}`);
+
+    try {
+        var result = await dataService.write(data);
```

Comment:
```suggestion
        const result = await dataService.write(data);
```
we live in the "var considered harmful" future now :)
either `const` or `let` is usually¹ better than `var` these days

⎯
¹ terms and conditions apply